packages/web/src/store/tipping/sagas.ts

```diff
@@ -97,6 +99,23 @@ function* overrideSupportingForUser({
   const supportingMap = yield* select(getOptimisticSupporting)
   const supportingForSender = supportingMap[sender.user_id] ?? {}
 
+  /**
+   * If sender was not previously supporting receiver, then
```

Comment:
nit: code seems to say, if receiver was not previously supporting sender?

Reply:
@piazzatron I don't think so - `supportingForSender` is a list of who the sender supports, and it's checking if the receiver is there

Reply:
u right

Reply:
yup what marcus said